eProctoring/user/views.py

```
@@ -1,4 +1,71 @@
+            messages.success(request, "You have successfully logged in.")
+            return redirect("profile") # Redirect to the profile page after successful login
+        else:
+            messages.error(request, "Invalid username or password.")
+    
+    context = {'title': 'Login'}
+    return render(request, 'registration/login.html', context)
+
+
+
+
+def register_view(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `register_view`

(ANN201)